infra/feast-operator/config/samples/v1alpha1_featurestore_all_services_default.yaml

```diff
@@ -1,14 +1,12 @@
 apiVersion: feast.dev/v1alpha1
 kind: FeatureStore
 metadata:
-  name: sample-all-default
+  name: sample-all-services
 spec:
   feastProject: my_project
   services:
-    onlineStore:
-      image: 'feastdev/feature-server:0.40.0'
-    offlineStore:
-      image: 'feastdev/feature-server:0.40.0'
+    onlineStore: {}
+    offlineStore: {}
+    ui: {}
```

Comment:
I believe `ui` component is not part of this release 0.43, not sure this will make any issue

Reply:
good catch!

Reply:
fixed